python/Ganga/Runtime/Repository_runtime.py

```diff
@@ -134,7 +134,15 @@ def bootstrap():
         registry.startup()
         logger.debug("started " + registry.info(full=False))
         if registry.name == "prep":
-            registry.print_other_sessions()
+            # Get a list of strings describing other running sessions
+            other_sessions = registry.get_other_sessions()
```

Comment:
No because then you spam the user with un-needed warnings... I'll change the PR...